app/templates/check_results.html

```diff
@@ -259,6 +259,55 @@ <h3>
                             </table>
                         </div>
                     </div>
+
+                    <div class="card p-1 bg-light border rounded-3 mb-3">
+                        {{ card_header("TLS: Mail Exchange", result.domain.tls.mail_transfer) }}
+                        <div class="card-body">
+                            {% for mail_exchange in result.domain.tls.mail_transfer.mail_exchanges %}
+                            <table class="table">
+                                <tr>
+                                    <td class="label">{% trans %}Host{% endtrans %}</td>
+                                    <td><code>{{ mail_exchange.host }}</code></td>
```

Comment:
![image](https://github.com/CERT-Polska/mailgoose/assets/1233067/3a9ad57f-2ace-4563-8f35-b089d8328fcf)

works with netcat, why does mailgoose show it can't connect?

Reply:
OVH mail servers take longer thank 5s to respond :man_shrugging: